automation_tools/__init__.py

```diff
@@ -865,6 +871,8 @@ def downstream_install(admin_password=None, run_katello_installer=True):
 
     # Install required packages for the installation
     run('yum install -y katello')
```

Comment:
This will run in case of version 6.2 also, please put this in else clause below.